### PR TITLE
Project import generated by Copybara.

### PR DESCRIPTION
**Pull request by trav3711**
PiperOrigin-RevId: 441020069

### CODE REVIEW

**Comment by trav3711** [comment]
Can we force this through? The things it is failing on are Instance Validator linter errors which weren't thrown by the g3 linter...

**Comment by charbull** [comment]
@trav3711 please fix linter

File: .github/workflows/python_tools.yml [hunk 1]
```diff
@@ -51,6 +51,18 @@ jobs:
       run: |
        python -m unittest discover -p '*_test.py' --failfast -v
       working-directory: ./tools/explorer
+    #---------- Configuration scoring ----------#
```
Comment (charbull) [inline .github/workflows/python_tools.yml:54]: undo? keep the scorer after the validator since there is a dependency

**Review by charbull — CHANGES_REQUESTED** [review]
fix linter please

File: .github/workflows/python_tools.yml (hunk 1 above)
Comment (trav3711) [reply .github/workflows/python_tools.yml:54]: Ah ok, this is incorrect in g3 as well then. Will remove.

Comment (charbull) [reply .github/workflows/python_tools.yml:54]: did you remove? I am not sure if I am looking at the latest version?

**Review by charbull — CHANGES_REQUESTED** [review]
revert the github actions file please

**Review by charbull — APPROVED** [review]

**Pull request merged**